components/meter/meter-linear.js

```diff
@@ -86,7 +85,7 @@ class MeterLinear extends MeterMixin(RtlMixin(LitElement)) {
 				align-self: flex-end;
 			}
 
-			:host([dir="rtl"]) .d2l-meter-linear-primary-ltr {
+			.d2l-meter-linear-primary-ltr {
```

Comment:
Previously this would only have been applied when `dir="rtl"`... the direction would have been `ltr` anyway.